reports/Florida.md

```diff
@@ -35,6 +35,20 @@ id: fl-fortlauderdale-2
 * [CNN Interview](https://twitter.com/greg_doucette/status/1270479109982912514)
 
 
+### Police laugh and joke when they think their body cams are off | May 31st
+
+Body cam video released by Fort Lauderdale PD on Youtube shows police laughing and joking when they believe their cams are in "standby" mode and not recording. Policing experts who reviewed the footage say that the behavior is unprofessional and that the officers felt "comfortable about talking about turning the camera off." At 2:55, a cop can be heard saying "if you need it, give me a target." The Miami Herald quotes a retired Lieutenant Colonel as commenting, "If you're not seeing a threat, why are you asking for a threat? That would almost lead you to believe that they were indiscriminately firing."
```

Comment:
This looks pretty good, I would also just add back in that this is after having shot rubber bullets/ tear gas

Reply:
I've added this in. I split the first sentence into two for readability.